src/actions/__tests__/swapParent.ts

```diff
@@ -31,7 +34,11 @@ it('no-op if cursor is a root thought', () => {
     - b
      - c`
 
-  const steps = [importText({ text }), setCursor(['a']), swapParent()]
+  const steps = [
+    importText({ text }),
+    (state: State) => setCursor({ path: contextToPath(state, ['a'])! })(state),
```

Comment:
I generally dislike using `!` (non-null assertion operator) in TS as it's an unsafe escape hatch from the types we've set up, but I'm less worried in this case since we're only using it in the test suite. This comment is non-blocking and does not need to be addressed 👍

Reply:
Thanks, I will keep this in mind.